src/Backend/Modules/Pages/Domain/Page/Form/PageSettingsType.php

```diff
@@ -32,21 +33,19 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         );
         $builder->add(
             'publishOn',
-            DateTimeType::class,
+            DatePickerType::class,
```

Comment:
Is DateTimeType still used here somewhere? Otherwise you need to remove the use statement